scripts/ingest_smoke_test.py

```diff
@@ -0,0 +1,232 @@
+set up access token on command line
+ACCESS_TOKEN=`gcloud auth print-access-token`
+
+EXAMPLE
+Creates N new study/ies in staging
+python ingest_smoke_test.py --token=$ACCESS_TOKEN -n <number of studies>
+
+Uses existing study, 'createTest' (SCP88 in staging), uploads 8K data set
+# Usage conditions: must have VPN access to staging server
+#                   files being uploaded cannot already exist in createTest
+python ingest_smoke_test.py --token=$ACCESS_TOKEN --dev-run
+"""
+
+import argparse
+import subprocess
+from datetime import datetime
+
+DATA = {
+    # matrix has ~8K cells, ~20K genes (aka "mult-bcl" data set); SCP23 and SCP31 in staging
+    # baseline ingest for dense TBD
+    "small": {
+        "dense": "gs://fc-2f8ef4c0-b7eb-44b1-96fe-a07f0ea9a982/test_Data/scp-ingest-pipeline/launch_ingest_test_data/mult_bcl_out/mult_bcl_out.scp.expr.txt.gz",
+        "cluster": "gs://fc-2f8ef4c0-b7eb-44b1-96fe-a07f0ea9a982/test_Data/scp-ingest-pipeline/launch_ingest_test_data/mult_bcl_out/mult_bcl_out.scp.X_tsne.coords.txt",
+        "metadata": "gs://fc-2f8ef4c0-b7eb-44b1-96fe-a07f0ea9a982/test_Data/scp-ingest-pipeline/launch_ingest_test_data/mult_bcl_out/mult_bcl_out.scp.metadata.txt",
+    },
+    # matrix has ~18.7K cells, ~25K genes (aka "PolypScrape" data from Nasal polyps)
+    # baseline ingest for dense ~4.5m
+    "medium": {
+        "dense": "gs://fc-2f8ef4c0-b7eb-44b1-96fe-a07f0ea9a982/test_Data/scp-ingest-pipeline/test_data_for_production_ingest/nasal_polyps/20180921_PolypScrape_cleaned_data.txt.gz",
+        "cluster": "gs://fc-2f8ef4c0-b7eb-44b1-96fe-a07f0ea9a982/test_Data/scp-ingest-pipeline/test_data_for_production_ingest/nasal_polyps/20180921_PolypScrape_cleaned_tsnecoord.txt",
+        "metadata": "gs://fc-2f8ef4c0-b7eb-44b1-96fe-a07f0ea9a982/test_Data/scp-ingest-pipeline/test_data_for_production_ingest/nasal_polyps/alexandria_structured_metadata.txt",
+    },
+    # matrix has ~220K cells, ~16K genes from SCP947 in prod
+    # baseline ingest for dense ~25.5m
+    "large": {
+        "dense": "gs://fc-2f8ef4c0-b7eb-44b1-96fe-a07f0ea9a982/test_Data/scp-ingest-pipeline/launch_ingest_test_data/SCP947/All.exp.tsv",
+        "cluster": "gs://fc-2f8ef4c0-b7eb-44b1-96fe-a07f0ea9a982/test_Data/scp-ingest-pipeline/launch_ingest_test_data/SCP947/All.cluster.tsv",
+        "metadata": "gs://fc-2f8ef4c0-b7eb-44b1-96fe-a07f0ea9a982/test_Data/scp-ingest-pipeline/launch_ingest_test_data/SCP947/metadata.tsv",
+    },
+}
+
+
+def create_parser():
+    """Parse command line values
+    """
+    parser = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+
+    parser.add_argument(
+        '-n',
+        '--number',
+        help='number of studies to create for stress test',
+        type=int,
+        default=1,
+    )
+
+    parser.add_argument(
+        '--environment',
+        help='server to test against [development,staging,production]',
+        default='staging',
```

Comment:
As in our [public CLI](https://github.com/broadinstitute/single_cell_portal/blob/master/scripts/cli_parser.py#L50):

```suggestion
        help='server to test against',
        choices=['development', 'staging', 'production'],
        default='staging',
```

Using `choices` gives built-in validation, and includes the options in `--help` output.

Reply:
New to this feature, love it!  `38d94ae`